tests/test_bidi.py

```diff
@@ -60,34 +64,38 @@ async def get_open_context_id(websocket):
     resp = await read_JSON_message(websocket)
     assert resp['id'] == 9999
     [context] = resp['result']['contexts']
-    contextID = context['context']
-    return contextID
+    context_id = context['context']
+    return context_id
+
 
 async def send_JSON_command(websocket, command):
     await websocket.send(json.dumps(command))
 
+
 async def read_JSON_message(websocket):
     return json.loads(await websocket.recv())
 
+
 # Open given URL in the given context.
-async def goto_url(websocket, contextID, url):
+async def goto_url(websocket, context_id, url):
     # Send "browsingContext.navigate" command.
     command = {
         "id": 9998,
         "method": "browsingContext.navigate",
         "params": {
             "url": url,
-            "context": contextID}}
+            "context": context_id}}
     await send_JSON_command(websocket, command)
 
-    # Assert "DEBUG.Page.load" event emitted.
-    resp = await read_JSON_message(websocket)
-    assert resp["method"] == "DEBUG.Page.load"
+    # # Assert "DEBUG.Page.load" event emitted.
+    # resp = await read_JSON_message(websocket)
```

Comment:
Did you mean to comment this out? The description doesn't mention any real changes.

Reply:
removed